src/Pages/CollectionPage/CollectionPage.js

```diff
@@ -1,4 +1,41 @@
+import CollectionListItem from '../../Components/CollectionListItem/ListItem.js';
+import bro from '../../assets/images/bro.png';
+import { collection } from '../../data.js';
+import CollectionCard from '../../Components/CollectionCard/CollectionCard.js';
+import FloatingBtn from '../../Components/floatingBtn/floatingBtn.js';
+
 function CollectionPage() {
-  return <div>Collection Page</div>;
+  return (
+    <div className="flex h-full bg-lightGrey">
```

Comment:
remove this div since the bg-lightGrey is added to the page even without this 
add the flex class to the div bellow this